Source/Fuse.Common/UpdateManager.uno

```diff
@@ -106,25 +126,11 @@ namespace Fuse
 			
 			list.Insert(0,us);
 		}
-		
-		/*
-			Keeps a sorted list of deferred actions. This list is processed in parallel, thus
-			the add only scans back to PhaseDeferredActionsAt (the one currently being executed).
-		*/
-		public void AddDeferredAction( Action pu, IUpdateListener ul, int priority = 0 )
-		{
-			int at = PhaseDeferredActions.Count;
-			while( at - 1 > PhaseDeferredActionsAt && 
-				PhaseDeferredActions[at-1].priority > priority )
-				at--;
-				
-			PhaseDeferredActions.Insert(at, new UpdateAction{ action = pu, update = ul, priority = priority });
-		}
-		
-		public void ResetDeferredActions()
+
+		public void AddDeferredAction( Action pu, IUpdateListener ul, LayoutPriority priority = LayoutPriority.Layout)
 		{
-			PhaseDeferredActions.Clear();
-			PhaseDeferredActionsAt = -1;
+			var queue = PhaseDeferredActions[(int)priority];
+			queue.Enqueue(new UpdateAction{ action = pu, update = ul });
 		}
```

Comment:
This way, we make the API breakage a bit more graceful, so people can update their code on their own schedule.

We could also use some different priority here, if needed.